scripts/demo/mockserver/main.go

```
@@ -0,0 +1,187 @@
+	user := flag.String("user", "demo", "username the demo config authenticates as")
+	flag.Parse()
+
+	now := time.Now()
+	mux := http.NewServeMux()
+	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		handle(w, r, *user, now)
+	})
+
+	addr := "127.0.0.1:" + *port
+	log.Printf("mock Beeminder API listening on http://%s (user %q)", addr, *user)
+	if err := http.ListenAndServe(addr, mux); err != nil {
```

golangci-lint (2.12.2)

[medium] G114: Use of net/http serve function that has no support for setting timeouts

(gosec)